### PR TITLE
Dev/org page

### PR DESCRIPTION
**Pull request by sandeep-v1404**
### Functionality:
To make Organization & Team component , so that all organizations can use WorkTrolly.

### Solution:

- By Making CreateNewOrganization, OrgDetails, TeamDetails Components and Modified Login Component for users to Sign Up  using Email.
- By Implementing Backend of CreateNewOrg to store the Organization details in the Backend.
- Making Security Keys in Backend.

### Risk level:
- [X] high 
- [ ] medium
- [ ] low

### How to test:

- To get no errors at the backend. Update Node in your local system.
- Open a terminal
- Update Node with **sudo npm i -g n**  , **sudo n stable**
- Close the terminal . And open new Terminal . Check node version using **node -v** . It should be v14.15.* .
- ng serve
- firebase emulators:start
- Click Login and Login with Google & then Logout.
- Click Login and Click on **Create an Organization** Button.

### CODE REVIEW

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
upload logo is not working.

**Comment by sandeep-v1404** [comment]
> upload logo is not working.

Change the Permissions in your firebase console. Go to Storage and Change the rules to true..

**Review by shivamgoyal7 — COMMENTED** [review]
For me API is not working properly. It shows cors error
(I am trying to rectify issue)

**Review by shivamgoyal7 — COMMENTED** [review]
For me API is not working properly. It shows cors error
(I am trying to rectify issue)

**Comment by bighnesh0404** [comment]
> > upload logo is not working.
> 
> Change the Permissions in your firebase console. Go to Storage and Change the rules to true..

**Review by bighnesh0404 — COMMENTED** [review]
not able to upload the logo

**Review by shivamgoyal7 — APPROVED** [review]
It's working fine in node 14

**Comment by viv9k** [comment]
Resolve merge conflict and adapt your code with new theme implementation.

**Review by viv9k — APPROVED** [review]
ship it

File: src/assets/errorCodes.xml [hunk 1]
```diff
@@ -88,4 +88,24 @@
...
+    <SubComponent name="TEAM-DETAILS" code="10">
+      <SubSystem code="0000">
+        <InternalError code="00">
+          <Error code="02" description="SERVER_ERROR"></Error>
+        </InternalError>
+        <ExternalError code="01">
+          <Error code="01" description="VALIDATION_ERROR"></Error>
+        </ExternalError>
+      </SubSystem>
+  </SubComponent>
```
Comment (viv9k) [inline src/assets/errorCodes.xml:110]: These two are the subsystem of create new organization component and can be moved in here.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please make the required changes in error implementation.

**Review by bighnesh0404 — APPROVED** [review]

File: src/assets/errorCodes.xml (hunk 1 above)
Comment (sandeep-v1404) [reply src/assets/errorCodes.xml:110]: Algorithm should be Optimized For Sub Systems.

**Pull request merged**